docs/api/README.md

```diff
@@ -1083,16 +1083,19 @@ sidebar: auto
 
 ### optional
 
-- **Signature:** `optional(validation)`
+- **Signature:** `optional(validation, considerTrimmedEmptyString = false)`
 
 - **Arguments:**
 
   - `validation: Validation`
+  - `considerTrimmedEmptyString: boolean` (defaults to false)
```

Comment:
The defaults to false part can be left out since the default is specified in the signature. This is consistent across the docs.